SD_Card_Write/SD_Card_Write.ino

```diff
@@ -0,0 +1,57 @@
+#include <SD.h>
+#include <SPI.h>
+
+File rocketSDCard;
+
+void setup() {
+  // Open serial communications and wait for port to open:
+  Serial.begin(9600);
+  while (!Serial) {
+    ; // wait for serial port to connect. Needed for native USB port only
+  }
+
+
+  Serial.print("Initializing SD card...");
+
+  if (!SD.begin(28)) {
```

Comment:
28? There are only 14 digital pins (0-13). If it's connected to an analog pin I'd rather see e.g. `A0` for clarity.